packages/suite/src/views/dashboard/components/PortfolioCard/components/Exception/index.tsx

```diff
@@ -75,7 +75,7 @@ const Container = ({ title, description, cta }: ContainerProps) => {
             <Actions>
                 {actions.map(a => (
                     <Button
-                        key={a.label}
+                        key={a.label || 'TR_RETRY'}
```

Comment:
This is a bit weird. It would still throw a warning if there are multiple items without `label` see. And TS will allow you to do that if you want. But out of scope of this PR anyway.

Reply:
`a.label` is optional and default value is 'TR_RETRY' (line 85) so this fix solve `undefined` value only
there is also a possibility where you set multiple actions with the same `a.label` but i don't see a reason why to do that (same as using multiple retry buttons)